eea/dexterity/indicators/upgrades/configure.zcml

```diff
@@ -15,4 +15,16 @@
 
   </genericsetup:upgradeSteps>
 
+  <genericsetup:upgradeSteps
+    source="6.0"
+    destination="6.1"
+    profile="eea.dexterity.indicators:default">
+
+    <genericsetup:upgradeDepends
+       title="Add data_provenance and temporal_coverage Indexer"
+    />
+
+  </genericsetup:upgradeSteps>
```

Comment:
@dobri1408 There is no need for an upgrade step in this case. Only changes in xml files from profiles need an upgrade step.